### ISSUE TITLE
frappedocker_default network cannot be reached from outside

### ISSUE THREAD

**Issue by zenny** [frappe/frappe_docker#27]
Hi,

The 'docker-compose up -d' created it's own 'frappedocker_default' network, which cannot be reached from outside.

The ip forwarding is enabled.

```
zenny@docker:~/gitrepos/frappe_docker$ sysctl net.ipv4.conf.all.forwarding
net.ipv4.conf.all.forwarding = 1
```

```
zenny@docker:~/gitrepos/frappe_docker$ brctl show
bridge name	bridge id		STP enabled	interfaces
br-ae5636c26be3		8000.0242a6ae8ed6	no		veth06124fb
							veth16cda95
							veth30defb5
							veth68e54bf
							vethde6417a
br-c91e1ca6b0ed		8000.0242f181cf1f	no		
docker0		8000.02423fa8ced6	no		veth7e24060
```

The docker-compose created a separate bridge for frappe containers, frappedocker_default as follows:

```
zenny@docker:~/gitrepos/frappe_docker$ docker network ls
NETWORK ID          NAME                   DRIVER              SCOPE
defcad07cec7        bridge                 bridge              local
ae5636c26be3        frappedocker_default   bridge              local
7b4da0f2913d        host                   host                local
fd3b7cdb3542        none                   null                local
```

The frappedocker_network does not seem to bridged to docker0, and could be the reason that does not allow access to the containers connected to the frappedocker_default. The default 'bridge' created has an option, and maybe therefore, the containers (like urbackup in the screenshot at http://picpaste.com/IwBu2M5z.png) connected to the default bridge are readily accessible from outside!

```
        "Options": {
            "com.docker.network.bridge.default_bridge": "true",
            "com.docker.network.bridge.enable_icc": "true",
            "com.docker.network.bridge.enable_ip_masquerade": "true",
            "com.docker.network.bridge.host_binding_ipv4": "0.0.0.0",
            "com.docker.network.bridge.name": "docker0",
            "com.docker.network.driver.mtu": "1500"
        },


```

But frappedocker_default does not have above options as of below:

```
zenny@docker:~/gitrepos/frappe_docker$ docker network inspect frappedocker_default
[
    {
        "Name": "frappedocker_default",
        "Id": "ae5636c26be3e994bc149914e202c8d8b273745fc0b5d567c4b5563abfe27ac5",
        "Created": "2017-12-25T10:23:19.881608277+01:00",
        "Scope": "local",
        "Driver": "bridge",
        "EnableIPv6": false,
        "IPAM": {
            "Driver": "default",
            "Options": null,
            "Config": [
                {
                    "Subnet": "172.18.0.0/16",
                    "Gateway": "172.18.0.1"
                }
            ]
        },
        "Internal": false,
        "Attachable": false,
        "Ingress": false,
        "ConfigFrom": {
            "Network": ""
        },
        "ConfigOnly": false,
        "Containers": {
            "0f0d67fafcc19fd48b4fc5de8b509faa923c347eac14a2c3a6423b64904a5c91": {
                "Name": "redis-socketio",
                "EndpointID": "515bb681cc957ad93eabbab5f8b6fc543a0ed68464ae08cf2dc225324943b993",
                "MacAddress": "02:42:ac:12:00:03",
                "IPv4Address": "172.18.0.3/16",
                "IPv6Address": ""
            },
            "86aeaee4f8b6cd8eb0b439b650cfe3664c6fc71955a067f705881835c0ec779c": {
                "Name": "frappe",
                "EndpointID": "f575b3fcb4edfd95bda80f1f393f71476bbca5a5b1a2aa9d4a8b4399ad76dd2c",
                "MacAddress": "02:42:ac:12:00:06",
                "IPv4Address": "172.18.0.6/16",
                "IPv6Address": ""
            },
            "d743e83fb214d773169936fa20e1d5cf4f9ef47ef63f70aaa878e493c7ad2298": {
                "Name": "mariadb",
                "EndpointID": "c2e55f1d5f26faf8fc015f9d46c5f89dfe21dcf893cce276cfa229430ce54f4d",
                "MacAddress": "02:42:ac:12:00:05",
                "IPv4Address": "172.18.0.5/16",
                "IPv6Address": ""
            },
            "d8b6fd0aa57087475bb8a6883015569b93ee72e77aaca83618c7c54b3ccec642": {
                "Name": "redis-queue",
                "EndpointID": "1a51ecf3d0fb0034d26846225ebf62c57fdd39ffb83d35b61ba35b75c54dc4a5",
                "MacAddress": "02:42:ac:12:00:04",
                "IPv4Address": "172.18.0.4/16",
                "IPv6Address": ""
            },
            "db8f190c5920adb2d7f4261f8d730ff8d25cf4318ba96f6031e87172508a6838": {
                "Name": "redis-cache",
                "EndpointID": "5a967bb58f023759c5222cba14be8d9b2c5a2bcac6e0acf98ddadaec7ac6b988",
                "MacAddress": "02:42:ac:12:00:02",
                "IPv4Address": "172.18.0.2/16",
                "IPv6Address": ""
            }
        },
        "Options": {},
        "Labels": {}
    }
]
```
What is missing?

**Comment by xinliuswie** [comment]
Maybe the same problem with you. I have installed frappe-docker under the guide, everythings works well. But when I try to visit http://localhost:8000, there's nothing to response.

I have checked Windows port through commond "netstat -ano | findstr 8000" and it shows 8000 port is listening. But when I use "curl 127.0.0.1:8000" in the frappe container, it shows "connection is refused".
While "docker ps" shows that every container is running.

Does anyone have met this problem?
